.github/workflows/ci-astro-deploy.yml

```diff
@@ -0,0 +1,20 @@
+name: "Astro Deploy"
```

Comment:
How could we give more visibility if this job fails? Would it be an option to notify Slack..?

Could it also be one of the steps of the default CI pipeline so that we ensure things are working and see them failing when we make changes to the SDK?

Reply:
separate PR?? maybe we should create a GH issue